decentraland/unity-renderer
# fix: Mute scene SFX when Backpack, BiW Editor or Tutorial is open

## What does this PR change?


Fixes #571 

If we are on a scene with some music or SFX playing in background (ie: Genesis Plaza), if we open a feature with its own music (ie: **Backpack**, **BiW Editor** or **Tutorial**), we're listening 2 musics at the same time: the music of the feature and the background music of the scene.

Now, we mute any kind of sound related to the scene while we open these features.

## How to test the changes?



1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/Mute-scene-audio-when-Backpack-BiW-or-Tutorial-is-open&ENV=org
2. Once you are in the Genesis Plaza (or any other scene with background music), open any of these features: **Backpack**, **BiW Editor** or **Tutorial**.
3. Notice that only the music of the feature is playing and the scene-related is muted until you close the feature.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
